src/hostname/hostnamed.c

```diff
@@ -430,6 +430,31 @@ static int context_write_data_machine_info(Context *c) {
         return write_env_file_label("/etc/machine-info", l);
 }
 
+static const char * const dmi_blocklist[] = {
```

Comment:
Given this comes from the kernel sources, could you please add the URL of the source as a comment? Pointing to a git tag for the latest release, so that the link keeps working and we also know when we updated it.